### ISSUE TITLE
Border color: transparentize( lighten( $gray, 20% ), .5 )

### ISSUE THREAD

**Issue by nylen** [Automattic/wp-calypso#17770]
This color shows up a lot in the Calypso code, usually related to border styles:

```
~/a8c/code/wp-calypso $ ack 'transparentize.{0,20}lighten.{0,20}gray' | grep -o 'transparentize.*$' | sort | uniq -c
      1 transparentize( lighten( $gray, 15% ), .5 );
      2 transparentize( lighten( $gray, 20% ), 0.5 ),
      2 transparentize( lighten( $gray, 20% ), 0.5 );
      2 transparentize( lighten( $gray, 20% ), 0.5 ), 0 1px 2px lighten( $gray, 30% );
      1 transparentize( lighten( $gray, 20% ), .25 );
      1 transparentize( lighten( $gray, 20% ), .3 );
     22 transparentize( lighten( $gray, 20% ), .5 ),
     15 transparentize( lighten( $gray, 20% ), .5 );
      3 transparentize( lighten( $gray, 20% ), .5 ), 0 1px 2px lighten( $gray, 30% );
```

Is this an officially recommended color for borders?

If so, we move it into a named SCSS variable.  There are also lots of opportunities to clean up which elements actually provide these borders, because right now many sections are a complicated mess of borders and `box-shadow` and it is often tricky to figure out where a given line is coming from.

**Comment by nylen** [comment]
This would probably be a good issue for an OSS contributor to tackle.

**Comment by dsab123** [comment]
I can take this one on 👍

**Comment by dsab123** [comment]
ok, so here's where I'm at thus far. Would appreciate any feedback!

`transparentize( $gray-lighten-20, 0.5 )` evaluates to `rgba(200, 215, 225, 0.5)` evaluates to `#c7d7e1`.

☝️ is not an _officially_ recommended color according to https://dotcombrand.wordpress.com/colors-archive/, but I'm not sure it has to be since it's a function output, eh?

I did discover this, though, in `~/assets/stylesheets/shared/_colors.scss`:
`
$gray-lighten-10: $muriel-gray-200; // #a8bece
$gray-lighten-20: $muriel-gray-100; // #c8d7e1
$gray-lighten-30: $muriel-gray-0; // #e9eff3
`

so we're working with #c8d7e1 and #c7d7e81, which are virtually indistinguishable from each other according to my little eyes.

I'm not sure if it'd be better to just use the `$gray-lighten-20` since it's already defined.

**Comment by dsab123** [comment]
`
$gray-lighten-10: $muriel-gray-200; // #a8bece

$gray-lighten-20: $muriel-gray-100; // #c8d7e1

$gray-lighten-30: $muriel-gray-0; // #e9eff3
`

to make the lines explicit

**Comment by flootr** [comment]
Hey @dsab123 thanks for looking into this. We recently began moving all our SASS variables to CSS custom properties which would be desirable here as well. Which means we probably don't want to use the `$gray-lighten-*` variables anymore.

We have a new way to define transparency with colors which is the following:

```css
.arbitrary-class {
  /* --arbitrary-color-rgb is derived from hex-to-rgb( some-hex-value ) */
  background-color: rgba( var( --arbitrary-color-rgb ), 0.3 );
}
```

You can find a few examples in [assets/stylesheets/shared/_color-schemes.scss](https://github.com/Automattic/wp-calypso/blob/master/assets/stylesheets/shared/_color-schemes.scss).

So, ideally we want to migrate any usage of `transparentize( ... )` to use the new way from above.

As an additional sidenote we do have a CSS custom prop for borders which we can make use of as well (https://github.com/Automattic/wp-calypso/blob/master/assets/stylesheets/shared/_color-schemes.scss#L185-L187).

Hope that helps and let me know if you have any further questions. Happy to help.

**Comment by dsab123** [comment]
I haven't forgotten about this issue! I've been busy with some classwork and projects. I should be freed up enough by Friday to continue working on this.

**Comment by dsab123** [comment]
Ok, finallllly making some headway into this!

There are a few comments that are incorrect, which tripped me up a bit, like this one (which I linked to in earlier comments), which I will update in my forthcoming commit:

```
$gray-lighten-20: $muriel-gray-100; // #c8d7e1
```

I've determined that `transparentize( $gray-lighten-20, 0.5 )` maps to `rgba( var( --color-neutral-700-rgb ), 0.5 )`.

I'll begin replacing all references to `transparentize()` accordingly 👍

**Comment by flootr** [comment]
@dsab123 `--color-neutral-700` is a much darker gray than `$gray-lighten-20` so I'm not sure that's a valid replacment. Let's ask @drw158 for feedback on which colors to use. As far as I can see we have the following occurrences of `$gray-lighten-20` in `transparentize( ... )` functions:

```
transparentize( $gray-lighten-20, 0.5 )
transparentize( $gray-lighten-20, 0.3 )
```

**Comment by flootr** [comment]
@dsab123 I created a PR to address those here https://github.com/Automattic/wp-calypso/pull/31067 - let me know what you think.

**Comment by drw158** [comment]
Yes, I think `gray-lighten-20` should be equivalent to `--color-neutral-100` in the new color palette.

**Comment by dsab123** [comment]
thanks guys, must have missed something when going from `$gray-lighten-20` to the css vars.

**Comment by noahtallen** [comment]
This was resolved in #31067
